visualizer/main.js

```diff
@@ -15,7 +15,7 @@ const drawUi = () => {
   document.body.classList.add('is-font-loaded')
 
   menu.on('toggle-theme', function () {
-    document.documentElement.classList.toggle('light-theme')
+    document.documentElement.classList.toggle('toggle-theme')
```

Comment:
why this name change?

Reply:
@goto-bus-stop - Makes more sense this way. The button is not always to enable the light theme. Sometimes its to enable the dark theme. Following this logic we are toggling the theme. 

Its also to fall in line with the "toggle-grid" button which isn't called "toggle-four-boxes" for example. :smile:

Reply:
But this class is applied to the body to enable the light theme, right? :thinking:

Reply:
@goto-bus-stop - Fair enough! Can't argue with that. The toggle-theme will be light-theme again. :smile: